src/ch/imedias/rsccfx/view/RsccRequestPresenter.java

```diff
@@ -60,6 +83,26 @@ private void attachEvents() {
   }
 
   /**
+   *  Events for the supporter buttons, has to be called by createNewSupporterBtn too.
+   */
+  private void attachButtonEvents() {
+    for (int i = 0; i < buttons.size(); i++) {
+      Button b = buttons.get(i);
+      int finalI = i;
+      if (i < supportAddresses.size()) {
+        b.setOnMouseClicked(event ->
+            new SupporterAttributesDialog(supportAddresses.get(finalI).getDescription(),
+                                          // TODO split address into address and port
```

Comment:
This ToDo still relevant?
I'm not yet finished with this review... have to do something else :)